Pipfile

```diff
@@ -13,6 +13,9 @@ flake8 = "*"
 mypy = "*"
 build = "*"
 isort = "*"
+pytest-cov = "*"
+tox = "<4.0"
```

Comment:
[ask] Why is this dependency strictly specified?

Reply:
As of `tox` version `4.0` ithe _entry point name_ is `tox4` and not `tox`.

From https://github.com/tox-dev/tox/blob/4.0.0b1/docs/changelog.rst

> ... 4.0.0a1 is a complete rewrite of the package

When it stabilizes, we can add a `>=4.0` and replace `tox` cli cmd with `tox4` cli cmd everywhere.

Reply:
Understood! Thanks!